plugins/ign-docker-env.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 David Rosa
+# Licensed under the Apache License, Version 2.0
+
+
+# gzdev docker-env --ignition citadel
+# gzdev docker-env --ignition dome --linux-distro ubuntu:focal
+# gzdev docker-env --ignition dome --docker-args '-v /foo:/foo'
+
+"""
+Usage:
+    gzdev ign-docker-env IGN_RELEASE
+                [--linux-distro <linux-distro>]
+                [--docker-args DOCKER_ARGS]
+    gzdev ign-docker-env -h | --help
+    gzdev ign-docker-env --version
+
+Options:
+    -h --help                   Show this screen
+    --version                   Show gzdev's version
+    --linux-distro=linux-distro Linux distibution to use in docker env
+    --docker-args DOCKER_ARGS   Extra arguments to pass to docker
+"""
+
+from docopt import docopt
+from subprocess import check_call
+from sys import stderr
+import os
+
+
+ROCKER_CMD = ['rocker', '--x11', '--user']
+
+
+def detect_nvidia():
+    return [x for x in os.listdir('/dev/') if x.startswith('nvidia')] != []
+
+
+def _check_call(cmd):
+    print('')
+    print("Invoking '%s'" % ' '.join(cmd))
+    print('')
+
+    try:
+        check_call(cmd)
+    except Exception as e:
+        print(str(e))
+
+
+def error(msg):
+    print(f"\n {msg} \n", file=stderr)
+    exit(-1)
+
+
+# TODO: use a yaml file to get this information
+def default_distro_by_ignition(ignition_release):
+    if ignition_release == 'dome':
+        return 'ubuntu:focal'
+    elif ignition_release == 'citadel':
+        return 'ubuntu:bionic'
+    else:
+        error(f"Unknown ignition release {ignition_release}")
```

Comment:
Should we also support edifice here? `ign-rocker` _should_ support citadel, dome and edifice.

Reply:
You are right, added 8109d31